nifi-docs/src/main/asciidoc/user-guide.adoc

```diff
@@ -834,6 +834,7 @@ image:connection-settings.png["Connection Settings"]
 The Connection name is optional. If not specified, the name shown for the Connection will be names of the Relationships
 that are active for the Connection.
 
+===== FlowFile Expiration
 File expiration is a concept by which data that cannot be processed in a timely fashion can be automatically removed from the flow.
```

Comment:
I think the first word here should be *FlowFile* -- *FlowFile expiration is a concept...*.